app/assets/javascripts/select-kit/components/user-tracking-dropdown.js.es6

```diff
@@ -0,0 +1,76 @@
+import DropdownSelectBox from "select-kit/components/dropdown-select-box";
+
+export default DropdownSelectBox.extend({
+  pluginApiIdentifiers: [""],
+  classNames: ["user-tracking", "user-tracking-dropdown"],
+  nameProperty: "label",
+  allowInitialValueMutation: false,
+
+  computeHeaderContent() {
+    let content = this._super(...arguments);
+    if(this.get("user.ignored")) {
```

Comment:
I thought prettier was supposed to catch this but we want spaces before `(` - can you confirm this code was linted?

Reply:
Yes, sure, just wanted to leave linting, etc out since this is a draft. I am more interested in overall approach validation 👍